chunky/src/java/se/llbit/math/Octree.java

```diff
@@ -836,8 +836,8 @@ public OctreeImplementation getImplementation() {
     return implementation;
   }
 
-  public static void addImplementationFactory(String name, ImplementationFactory factory) {
-    factories.put(name, factory);
+  public static void addImplementationFactory(ImplementationFactory factory) {
```

Comment:
This breaking change is bad and we should find a way to work around this (maybe a backward-compatible `ImplementationFactory` and a new `RegisterableImplementationFactory`).